### PR TITLE
Added services shapes and cleanup

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[Added services shapes and cleanup](https://trello.com/c/LKG8Dzxg/807-added-services-shapes-and-cleanup)

**Comment by huwshimi** [comment]
QA:
- add an app
- in the list of apps hover over the added app and the circle should highlight
- click on the app in the list and the circle should get selected

File: jujugui/static/gui/src/app/init/component-renderers-mixin.js [hunk 1]
```diff
@@ -94,13 +94,12 @@ const ComponentRenderersMixin = superclass => class extends superclass {
         visible={db.services.size() > 0}>
         <AddedServicesList
           changeState={this._bound.changeState}
-          findRelatedServices={db.findRelatedServices.bind(db)}
-          findUnrelatedServices={db.findUnrelatedServices.bind(db)}
           hoveredId={hoveredId}
-          hoverService={ServiceModule.hoverService.bind(ServiceModule)}
-          panToService={ServiceModule.panToService.bind(ServiceModule)}
-          services={db.services}
-          updateUnitFlags={db.updateUnitFlags.bind(db)} />
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/init/component-renderers-mixin.js]: A bunch of these props were passed in but not used.

File: jujugui/static/gui/src/app/init/component-renderers-mixin.js [hunk 2]
```diff
@@ -94,13 +94,12 @@ const ComponentRenderersMixin = superclass => class extends superclass {
...
+          serviceModule={shapeup.addReshape({
```
Comment (frankban) [inline jujugui/static/gui/src/app/init/component-renderers-mixin.js]: Can't you use fromShape here?
`serviceModule={shapeup.fromShape(ServiceModule, AddedServicesList.propTypes.serviceModule)}` ?

Comment (huwshimi) [reply jujugui/static/gui/src/app/init/component-renderers-mixin.js]: Yes! Thanks, fixed.

**Comment by huwshimi** [comment]
Thanks @frankban, shapeFrom added. :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2897

**Pull request merged**